src/uint256.cpp

```diff
@@ -19,7 +19,11 @@ base_blob<BITS>::base_blob(const std::vector<unsigned char>& vch)
 template <unsigned int BITS>
 std::string base_blob<BITS>::GetHex() const
 {
-    return HexStr(std::reverse_iterator<const uint8_t*>(m_data + sizeof(m_data)), std::reverse_iterator<const uint8_t*>(m_data));
+    uint8_t m_data_rev[WIDTH];
```

Comment:
Correct me if I'm wrong, but isn't this duplicating memory usage?

Reply:
Not really: is is a local variable, all this does is create a small temporary buffer on the stack.